pomerium/ingress_annotations.go

```diff
@@ -258,3 +302,17 @@ func b64(secret *corev1.Secret, annotation, key string) (string, error) {
 	}
 	return base64.StdEncoding.EncodeToString(data), nil
 }
+
+func mergeMaps(src map[string][]byte, dst *map[string]string) error {
+	if dst == nil {
```

Comment:
dst should never be nil. I think what you want is `if *dst == nil` and then later `*dst = m`